.github/workflows/ci.yml

```
@@ -69,21 +56,31 @@ jobs:
 
       # CI TESTING
       - name: Run CI tests
```

```suggestion
      - name: Run software testsuite
```
Reading the whole thing today, seems clear to me to specify that what we are running is the testsuite. Here and in the debug step.